src/components/Survey/PetTypeQuestion/index.tsx

```diff
@@ -0,0 +1,49 @@
+'use client'
```

Comment:
Think for the components nested in 'Survey' component u actually don't need to declare 'use client' alr since Survey is client component